openwisp_radius/tests/test_api.py

```diff
@@ -1113,7 +1107,14 @@ def test_register_201(self):
         self.assertTrue(user.is_active)
 
     @mock.patch.object(
-        app_settings, 'OPTIONAL_REGISTRATION_FIELDS', optional_settings_params
+        app_settings,
+        'OPTIONAL_REGISTRATION_FIELDS',
+        {
+            'first_name': 'disabled',
+            'last_name': 'allowed',
+            'birth_date': 'disabled',
+            'location': 'mandatory',
+        },
```

Comment:
@NoumbissiValere unless the attribute `optional_settings_params` would be reused in other tests, there's no reason to store it as a class attribute, better pass it directly here. It makes the test declaration faster to read.

Reply:
Ok @nemesisdesign